README.md

```diff
@@ -0,0 +1,32 @@
+# OpenAstronomy conda channel
+
+## Installing packages
+
+To add this channel to your list of default conda channels:
+`conda config --add channels openastronomy`
+
+To install a single package from this channel:
+`conda install -c openastronomy <package>`
+
+## List of packages
+
+The list of packages on this channel is at
+https://anaconda.org/openastronomy/packages
+
+It includes all of the packages on these conda channels:
+
++ `astropy`: channel for [astropy](http://astropy.org) [affiliated packages](http://astropy.org/affiliated)
+
+## How can I add my package?
```

Comment:
@eteq At the moment the transfer is manual (in the sense that I ran a script locally to do the initial copying). I've been meaning to open a PR to conda-build-all to allow cross-channel copying, but it won't happen for a couple more weeks at least.

It is tempting to move all of the building over here because it has a separate travis/appveyor build queue, but that gets a little complicated for maintaining a separate affiliate channel.

Can you open an issue or PR for this?